src/gei/Commands/MigrateRepoCommand.cs

```diff
@@ -196,6 +265,83 @@ public async Task Invoke(string githubSourceOrg, string adoSourceOrg, string ado
             }
         }
 
+        private async Task<(string GitArchiveUrl, string MetadataArchiveUrl)> GenerateAndUploadArchive(
+          string ghesApiUrl,
+          string githubSourceOrg,
+          string sourceRepo,
+          string azureStorageConnectionString,
+          bool noSslVerify = false
+          )
```

Comment:
nit: it's not customary in c# to move the closing paran into a new line:

```suggestion
          bool noSslVerify = false)
```